tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -69,6 +78,12 @@ internal static Scope SendStartInvocation(ILambdaExtensionRequest requestBuilder
 
     internal static void SendEndInvocation(ILambdaExtensionRequest requestBuilder, Scope scope, bool isError, string data)
     {
+        if (string.IsNullOrEmpty(ApiKey))
+        {
+            Log("No API key configured, not calling the Extension at the end of invocation", debug: false);
```

Comment:
Given the problem is "too much logging", do you _want_ to log here 🤔

Reply:
I changed it to only log once, which should be ok, since we always want to notify the user this!